Library/Homebrew/rubocops/shell_cmd.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require "rubocops/extend/formula"
+
+module RuboCop
+  module Cop
+    module FormulaAudit
+      # This cop audits shell commands `system` and `Utils.popen`.
+      class ShellCmd < FormulaCop
```

Comment:
This could live in `Lines` instead, if desired.

Reply:
This looks right to me for all three related changes. I'll move them over.